### ISSUE TITLE
[Machine Learning] Dropout Regularization to handle overfitting in deep learning models. 

### ISSUE THREAD

**Issue by francis966** [section-engineering-education/engineering-education#5314]
#### Proposed title of article
[Machine Learning] Dropout Regularization to handle overfitting in deep learning models. 

#### Proposed article introduction
Overfitting and underfitting are common phenomena in the field of machine learning and deep learning. The techniques used to tackle the overfitting problem is called dropout regularization.

Overfitting happens when a model perfectly learns the detail and noise in the training data, this will have a negative impact on the model's performance when it comes to testing data or when the model is exposed to a new environment. The model has a good performance on the training set, but when the model is exposed to a new environment it will give wrong predictions.

Underfitting occurs when the model can neither learn from the training data nor make predictions when exposed to a testing dataset or new environment. This model underperforms both in training and in testing when making predictions.

In this tutorial, we will build a deep learning model using TensorFlow. We will then use the dropout regularization technique to handle underfitting and overfitting. This ensures we have a model that has a good performance on both the training dataset and the testing dataset.
#### Key takeaways 
1. Overfitting vs underfitting.
2. Impacts of overfitting and underfitting model.
3. Building a deep learning model using TensorFlow.
4. Implementing Dropout Regularization.
5. Improving the model performance using Dropout Regularization.

#### Article quality
This is tutorial is unique because we will address both underfitting and overfitting in the deep learning model using a custom model. We will build a custom model using TensorFlow to help explain these problems. We will implement Dropout Regularization so that we can improve the model performance during the training and testing phase. This is a practical approach and a reader can easily follow from scratch. 

#### References
N/A

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

**Comment by LinusMuema** [comment]
good topic @francis966  - approved. Thanks for submitting your article and we look forward to reviewing and hopefully publishing it. I’m sure our readers will enjoy and find your {article type} very beneficial.
